### DISCUSSION TITLE
feature: avoid notification fatigue

### DISCUSSION THREAD

**Discussion by eyalk11** [folke/noice.nvim#834] [Q&A]
### Did you check the docs?

- [X] I have read all the noice.nvim docs

### Is your feature request related to a problem? Please describe.

Yes, some messages appear a  lot when ideally would have appear only once per buffer say. Like autosave notification. 
Most of them you don't want at all (solved), but some you want , but not too often. 

### Describe the solution you'd like

Specify using routes that a certain filter is fatigue -> display only once per buffer / 30 minutes / something. 

### Describe alternatives you've considered

None

**Comment by b0ae989c** [comment]
Could you give a concrete example of repeated notifications? Maybe I can come up with a quick fix.
